### PR TITLE
fix: Show Valid Message In Recent Transaction with reference to issue #49

### PR DESCRIPTION
**Pull request by sagar15795**
Changes

- Added Error view in fragment_recent_transactions.xml
- Added private method to show error view in RecentTransactionsFragment class
- remove inflateRecentTransactionList method and used notifyDataSetChanged method to update RecyclerView Adapter Data

### CODE REVIEW

**Comment by sagar15795** [comment]
@therajanmaurya Please review it.

**Comment by sagar15795** [comment]
@therajanmaurya Please review it

File: app/src/main/res/layout/fragment_recent_transactions.xml [hunk 1]
```diff
@@ -4,11 +4,41 @@
     android:layout_width="match_parent"
     android:layout_height="match_parent">
 
-    <android.support.v7.widget.RecyclerView
-        android:id="@+id/rv_recent_transactions"
-        android:layout_width="wrap_content"
-        android:layout_height="0dp"
-        android:layout_centerHorizontal="true"
-        android:layout_weight="1"/>
+    <LinearLayout
```
Comment (therajanmaurya) [inline app/src/main/res/layout/fragment_recent_transactions.xml:7]: There no hierarchy of using xml view. bad practice.
Please look this as an example : https://github.com/openMF/android-client/blob/master/mifosng-android/src/main/res/layout/fragment_client.xml

File: app/src/main/java/org/mifos/selfserviceapp/ui/fragments/RecentTransactionsFragment.java [hunk 2]
```diff
@@ -126,27 +138,43 @@ public void onItemLongPress(View childView, int position) {
...
     @Override
     public void showRecentTransactions(List<Transaction> recentTransactionList) {
-        this.recentTransactionList = recentTransactionList;
-        inflateRecentTransactionList();
+        if (recentTransactionList.size() > 0) {
+            this.recentTransactionList.clear();
+            this.recentTransactionList.addAll(recentTransactionList);
+            recentTransactionsListAdapter.notifyDataSetChanged();
+        } else {
+            setErrorView(true, getString(R.string.msg_no_transaction), R.drawable
```
Comment (therajanmaurya) [inline app/src/main/java/org/mifos/selfserviceapp/ui/fragments/RecentTransactionsFragment.java:154]: It's not an error that there is no transaction to show.

**Comment by therajanmaurya** [comment]
Fixed in #88

**Pull request closed without merging**